AirCasting/MainTabBarView.swift

```diff
@@ -99,6 +101,14 @@ private extension MainTabBarView {
                 Image(homeImage)
             }
             .tag(TabBarSelection.Tab.dashboard)
+            .overlay(
+                Group{
+                    if sessions.count > 1 && selectedSection.selectedSection == .following {
```

Comment:
the fact that we're using a single view to show all sections is nuts. We need to move out of it and remove all of such if statements...